deploy/examples/create-external-cluster-resources.py

```diff
@@ -925,6 +934,7 @@ def get_rbd_provisioner_caps_and_entity(self):
             rbd_pool_name = self._arg_parser.rbd_data_pool_name
             alias_rbd_pool_name = self._arg_parser.alias_rbd_data_pool_name
             k8s_cluster_name = self._arg_parser.k8s_cluster_name
+            rados_namespace = self._arg_parser.rados_namespace
```

Comment:
Is the rados_namespace an optional argument?

Reply:
yes it should be an optional one.